agent/app/agent_windows.go

```diff
@@ -54,6 +55,7 @@ func newHandler(agent agent) *handler {
 // (the Windows goroutine will exit based on messages from Windows while the agent goroutine exits if the agent exits)
 // and then cancel the other.  Once everything has stopped running, this function returns and the process exits.
 func (h *handler) Execute(args []string, requests <-chan svc.ChangeRequest, responses chan<- svc.Status) (bool, uint32) {
+	defer seelog.Flush()
```

Comment:
What problem is this solving?

Reply:
I was seeing logs not get flushed sometimes.  This helped make sure they got flushed when we return from this function.